changelogs/unreleased/3667-srinivasa-vasu

```diff
@@ -0,0 +1 @@
+update storage provider setup link references
```

Comment:
It's a doc change, changelog is no needed.

Reply:
sure, removed